src/store/styleguide/slot.ts

```diff
@@ -0,0 +1,37 @@
+import { Store } from '../../store/store';
+
+export class Slot {
```

Comment:
Could you please add documentation here...
* what slots are
* how they are related to page elements
* hints to React's children and other framework-specific concepts
Thanks!